### DISCUSSION TITLE
C++ error not shown in red coloured text in the terminal

### DISCUSSION THREAD

**Discussion by karthick191999** [microsoft/vscode-cpptools#7206] [Q&A]
I had this confusion on why my VS code doesnt show the error(in the terminal) in the red text. Earlier it was showing me the error that it had found out in the red colour, but now it is showing me the error in just plain white which matches with the terminals text and makes it difficult for me to read and follow. Any ideas on why ??
PS : I'm not talking abt red squiggly lines, that seems to work tho and I'm working on windows

**Comment by sean-mcmanus** [comment]
Are you using the "cppbuild" task type? That task type creates its own terminal which doesn't support the red text colors. You could file an issue to request that. You could also set the task type to something else like "shell".

**Reply by karthick191999** [reply]
Cool yes @sean-mcmanus I'm using the cppbuild task type. Could you pls elaborate on the setting of task type to shell ?

**Reply by sean-mcmanus** [reply]
The tasks.json entry has a "type" that you could change from "cppbuild" to "shell". And/or you could file a feature request for us to add colors to the cppbuild task's shell (although I'm not sure if that's possible).

**Reply by karthick191999** [reply]
Ah nice i'll look into it and really thanks for your kind reply, I'll try filing for the feature request :) then.

**Reply by marpe** [reply]
I had this same problem and simply changing the type to `shell` worked. Don't know why this isn't the default when the tasks.json file is being generated.

**Reply by sean-mcmanus** [reply]
@marpe This should be fixed with 1.7.0-insiders2 (with the "cppbuild" task type).
